component.yaml

```diff
@@ -0,0 +1,153 @@
+platform: Common
+buildPlatform: ReactNative, JavaKotlin, Flutter, ObjectiveCSwift
```

Comment:
Remove ReactNative and Flutter platforms from the supported platforms.
- **[final]** Remove ReactNative specific information from  README.
- https://linear.app/appcircle/issue/BE-3930/sauce-labs-android-ui-test-analysis-and-turn-the-component#comment-30986cd6